api/src/main/java/org/openmrs/api/impl/OrderServiceImpl.java

```diff
@@ -92,6 +98,48 @@ public synchronized Order saveOrder(Order order, OrderContext orderContext) thro
 	}
 
 	/**
+	 * @see org.openmrs.api.OrderService#saveOrders(Set)
+	 */
+	public synchronized Set<Order> saveOrders(Set<Order> orders) {
```

Comment:
This is meant to save an order group right? I would think that this should be something like saveOrderGroup and takes an OrderGroup object